src/main/resources/static/src/views/Users/EditUsers.js

```diff
@@ -0,0 +1,189 @@
+                                                   placeholder="lastName"/>
+                                        </Col>
+                                    </FormGroup>
+
+                                    <FormGroup row>
+                                        <Col md="2">
+                                            <Label>Registration Date</Label>
+                                        </Col>
+                                        <Col xs="12" md="4">
+                                            <InputGroup>
+                                                <Input value={this.state.users.registrationDate} type="text"
+                                                       name="registrationDate" placeholder="DD/MM/YYYY hh:mm"
+                                                       onChange={this.handleDateChange}/>
+                                                <InputGroupAddon className={this.state.registrationDate ?
+                                                    "fa fa-calendar-check-o" : "fa fa-calendar-times-o"}/>
+                                            </InputGroup>
+                                        </Col>
+                                    </FormGroup>
+
+                                    <FormGroup row>
+                                    <Col md="2">
+                                        <Label>Role</Label>
+                                    </Col>
+                                    <Col xs="12" md="4">
+                                        <Input value={this.state.users.userRole} onChange={this.handleChange}
+                                               type="select" name="userRole"
+                                               placeholder="Role">
+                                            <option>ADMIN</option>
+                                            <option>USER</option>
+                                            <option>MODERATOR</option>
+                                        </Input>
+                                    </Col>
+                                </FormGroup>
+
+                                    <FormGroup row>
+                                        <Col md="2">
+                                            <Label>Status</Label>
+                                        </Col>
+                                        <Col xs="12" md="4">
+                                            <Input value={this.state.users.userStatus} onChange={this.handleChange}
+                                                   type="select" name="userStatus"
+                                                   placeholder="Role">
+                                                <option>DELETED</option>
+                                                <option>BANNED</option>
+                                                <option>ACTIVE</option>
+                                                <option>UNCONFIRM</option>
```

Comment:
UNCONFIRMED*. This should throw an exception.

Reply:
Fix